packages/core/src/service/ErrorReporter.ts

```diff
@@ -16,9 +16,11 @@ export class ErrorReporter {
 		severity = ErrorSeverity.Error,
 		info?: LanguageErrorInfo,
 	): void {
-		this.errors.push(
-			LanguageError.create(message, Range.get(range), severity, info),
-		)
+		if (message.trim() !== '') {
```

Comment:
Is there a case where we intentionally create an error with an empty message that should be ignored? I feel like if an error with empty message is created it's most certainly caused by a bug somewhere and we shouldn't just silently ignore them.

Reply:
I could throw an exception too, if you'd prefer that.

Reply:
That would probably be better (as long as we catch all of them during development lol)

Reply:
~~Just wrap all instances of err.report into a try catch~~